packages/android_intent/example/pubspec.yaml

```diff
@@ -0,0 +1,53 @@
+name: android_intent_example
+description: Demonstrates how to use the android_intent plugin.
+
+dependencies:
+  flutter:
```

Comment:
Done. I agree about these comments not being useful. Why do we have them in the plugin create? Presumably it moved over from the app create?